packages/mongodb-chatbot-ui/vite.config.component.cjs.ts

```diff
@@ -0,0 +1,52 @@
+import path from "path";
+import { defineConfig } from "vite";
+import react from "@vitejs/plugin-react";
+import { nodePolyfills } from "vite-plugin-node-polyfills";
+
+const entryPath = path.resolve(__dirname, "src/index.tsx");
+
+// https://vitejs.dev/config/
+export default defineConfig({
+  plugins: [
+    nodePolyfills({
+      exclude: [
+        "fs" as never, // Excludes the polyfill for `fs` and `node:fs`.
+      ],
+      globals: {
+        Buffer: true,
+        global: true,
+        process: true,
+      },
+      protocolImports: true,
+    }),
+    react(),
+  ],
+  build: {
+    outDir: "build/cjs",
```

Comment:
This is essentially what the `vite.config.component.ts` file was before code splitting was introduced, save for a few things like the `outDir` and `formats: ["cjs"],`